src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConstants.cs

```diff
@@ -0,0 +1,266 @@
+        }
+
+        public static class FormatStrings
+        {
+            /// <summary>
+            /// A string format used to poplate oauth authorize endpoint.
+            /// </summary>
+            public const string AuthorizeEndpoint = "https://{0}.superoffice.com/login/common/oauth/authorize";
+
+            /// <summary>
+            /// A string format used to poplate oauth token endpoint.
+            /// </summary>
+            public const string TokenEndpoint = "https://{0}.superoffice.com/login/common/oauth/tokens";
+
+            /// <summary>
+            /// A string format used to obtain user claims.
+            /// </summary>
+            /// <remarks>The final userinfo url contains protocol, host and tenant</remarks>
+            /// <example>https://sod.superoffice.com/Cust12345/api/v1/user/currentPrincipal</example>
+            public const string UserInfoEndPoint = "/{0}/api/v1/user/currentPrincipal";
+
+            /// <summary>
+            /// Format value used to construct <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
+            /// </summary>
+            public const string ClaimsIssuer = "https://{0}.superoffice.com";
+
+            /// <summary>
+            /// Format value used to construct well-known configuration endpoint.
+            /// </summary>
+            public const string ConfigurationEndpoint = "https://{0}.superoffice.com/login/.well-known/openid-configuration";
+        }
+
+        public static class PrincipalNames
+        {
+            /// <summary>
+            /// Type of user
+            /// </summary>
+            public const string UserType = "UserType";
+
+            /// <summary>
+            /// Associate name (e.g. logon name) for the user
+            /// </summary>
+            public const string Associate = "Associate";
+
+            /// <summary>
+            /// AssociateId for the user.
+            /// </summary>
+            public const string AssociateId = "AssociateId";
+
+            /// <summary>
+            /// Is this associate a person, and not a resource?
+            /// </summary>
+            public const string IsPerson = "IsPerson";
+
+            /// <summary>
+            /// Associate's person id
+            /// </summary>
+            public const string PersonId = "PersonId";
+
+            /// <summary>
+            /// Country id for the user.
+            /// </summary>
+            public const string CountryId = "CountryId";
+
+            /// <summary>
+            /// Country id for the user's home country. This is the default country id when creating new items.
+            /// </summary>
+            public const string HomeCountryId = "HomeCountryId";
+
+            /// <summary>
+            /// Company of the associate's person
+            /// </summary>
+            public const string ContactId = "ContactId";
+
+            /// <summary>
+            /// Associate's group_idx
```

Comment:
Group index?

Reply:
Yes, the comments were straight from out docs, but agree these could be cleared. I went back and fixed a few of these.